misc/formatter/modules/formatter-core/src/test/resources/declarations/import/source/import_declaration_10.bal

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
I know this license header is just for testing purposes. Having the correct header here wouldn't do any harm either :)

```suggestion
// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.com).
```